### PR TITLE
project-chatbot-evelyndelcarmen

### PR DESCRIPTION
**Pull request by EvelynDelCarmen**
https://project-chatbot-evelyndelcarmen.netlify.app/

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -27,25 +39,196 @@ const showMessage = (message, sender) => {
...
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage("Hi! Welcome. What's your name?", 'bot');
+};
+
+setTimeout(greetUser, 1000);
+
+const handleNameInput = (event) => {
+  console.log("event");
```
Comment (MikoZhu) [inline code/script.js]:
```javascript
console.log(event)
``` 
It's great to check the value, but the event here is not a string

File: code/script.js [hunk 2]
```diff
@@ -27,25 +39,196 @@ const showMessage = (message, sender) => {
...
+`;
+
+  const foodButtons = inputHandle.querySelectorAll('.send-btn');
+  console.log("foodbuttons");
+  foodButtons.forEach(button => {
+    button.addEventListener('click', (event) => {
+      const selectedfood = event.target.id;
+      showMessage(`Amazing! You've selected ${selectedfood}!`, 'bot');
+      foodChoice(selectedfood);
+
```
Comment (MikoZhu) [inline code/script.js]: Is it better to add one message for user response?
```javascript
showMessage(` ${selectedfood}! `,`user`)
```

File: code/script.js [hunk 3]
```diff
@@ -27,25 +39,196 @@ const showMessage = (message, sender) => {
...
+
+  foodOptions.forEach(option => {
+    console.log("option");
+    const optionElement = document.createElement('option');
+    optionElement.value = option;
+    optionElement.textContent = option;
+    dropdown.appendChild(optionElement);
+  });
+
```
Comment (MikoZhu) [inline code/script.js:133]: Do you mean that you want to check the option value here? 
```javascript
console.log(option) 
``` 
It is great to use the appendChild method here. 👍

File: code/script.js [hunk 4]
```diff
@@ -27,25 +39,196 @@ const showMessage = (message, sender) => {
...
+  confirmBtn.addEventListener('click', () => {
+    showMessage(`Thanks for confirming your order! You ordered ${foodChoice} (${selectedSize} size). Your food will soon be at your place.`, 'bot');
+    inputHandle.innerHTML = '';
+  });
+
+  cancelBtn.addEventListener('click', () => {
+    showMessage(`Your order has been canceled. Welcome back! Feel free to order food next time you are hungry.`, 'bot');
+    inputHandle.innerHTML = '';
+  });
+}
```
Comment (MikoZhu) [inline code/script.js:247]: There is a bug here. It shows the selectedSize is not defined. The second parameter "selectedSize" doesn't pass any value. It is undefined

**Review by MikoZhu — COMMENTED** [review]
Hi Evelyn!
Well done! You've finished the basic request for this project. The logic is great. 
My suggestion is to add user's messages which are more interactive with the bot.  There is one bug in the end, please check it again. 

I've learned a new method from you by using appendChild. Nice💐
Huadan(Miko)

File: code/script.js (hunk 4 above)
Comment (EvelynDelCarmen) [reply code/script.js:247]: Thank you Miko! 💫 I have now updated this repo.

**Review by EvelynDelCarmen — COMMENTED** [review]
Updated the project

**Pull request closed without merging**